packages/form/Readme.md

```diff
@@ -544,3 +545,179 @@ const Datepicker = require('./src').Datepicker;
 />
 
 ```
+
+### TelephoneNumber
+
+```
+const TelephoneNumber = require('./src').TelephoneNumber;
+<div>
+  <TelephoneNumber
+    id="tel-id-1"
+    name="tel-name"
+    label="TelephoneNumber"
+    description="Vul een telefoonnummer in"
```

Comment:
We're trying to offer our complete style guide in English, while the default values should be in Dutch.